.github/workflows/test.yml

```diff
@@ -0,0 +1,46 @@
+name: Test
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [ '1.20' ]
+        os: [ubuntu-latest, macos-latest]
+        libsql-server-release: [libsql-server-v0.24.32]
```

Comment:
can you also add a workflow_dispatch which lets us specify `libsql-server` version